BLDC.py

```diff
@@ -17,21 +18,24 @@ def __init__(self, pwmPin, directionPin, frequency):
             self.pwmCtrl = PWM.Signal(pwmPin, frequency)
 
     def setDirection(self, direction):
+        self.stop()
+        time.sleep(0.2)
```

Comment:
can you create a constant static variable (class variable) for the 0.2 sec sleep time?